runtime/litentry/src/lib.rs

```diff
@@ -885,6 +940,153 @@ impl pallet_bitacross::Config for Runtime {
+	pub BlockGasLimit: U256 = U256::from(
+		NORMAL_DISPATCH_RATIO * MAXIMUM_BLOCK_WEIGHT.ref_time() / WEIGHT_PER_GAS
+	);
+	pub PrecompilesValue: Precompiles = LitentryNetworkPrecompiles::<_>::new();
+	// BlockGasLimit / MAX_POV_SIZE
+	pub GasLimitPovSizeRatio: u64 = 4;
+}
+
+pub struct FindAuthorTruncated<T>(sp_std::marker::PhantomData<T>);
+impl<T: pallet_aura::Config> FindAuthor<H160> for FindAuthorTruncated<T>
+where
+	pallet_aura::Pallet<T>: FindAuthor<u32>,
+{
+	fn find_author<'a, I>(digests: I) -> Option<H160>
+	where
+		I: 'a + IntoIterator<Item = (ConsensusEngineId, &'a [u8])>,
+	{
+		if let Some(author_index) = pallet_aura::Pallet::<T>::find_author(digests) {
+			let authority_id =
+				<pallet_aura::Pallet<T>>::authorities()[author_index as usize].clone();
+			return Some(H160::from_slice(&authority_id.encode()[4..24]))
+		}
+
+		None
+	}
+}
+
+impl pallet_evm::Config for Runtime {
+	type FeeCalculator = TransactionPaymentAsGasPrice;
+	type GasWeightMapping = pallet_evm::FixedGasWeightMapping<Self>;
+	type WeightPerGas = WeightPerGas;
+	type BlockHashMapping = pallet_ethereum::EthereumBlockHashMapping<Self>;
+	type CallOrigin = pallet_evm::EnsureAddressTruncated;
+	type WithdrawOrigin = pallet_evm::EnsureAddressTruncated;
+	// From evm address to parachain address
+	type AddressMapping = pallet_evm::HashedAddressMapping<BlakeTwo256>;
+	type Currency = Balances;
+	type RuntimeEvent = RuntimeEvent;
+	type Runner = pallet_evm::runner::stack::Runner<Self>;
+	type PrecompilesType = Precompiles;
+	type PrecompilesValue = PrecompilesValue;
+	type ChainId = ChainId;
+	type OnChargeTransaction = OnChargeEVMTransaction<DealWithFees<Runtime>>;
+	type BlockGasLimit = BlockGasLimit;
+	type Timestamp = Timestamp;
+	type OnCreate = ();
+	type FindAuthor = FindAuthorTruncated<Runtime>;
+	// BlockGasLimit / MAX_POV_SIZE
+	type GasLimitPovSizeRatio = GasLimitPovSizeRatio;
+	type WeightInfo = weights::pallet_evm::WeightInfo<Runtime>;
+}
+
+parameter_types! {
+	pub const PostBlockAndTxnHashes: PostLogContent = PostLogContent::BlockAndTxnHashes;
+}
+
+impl pallet_ethereum::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type StateRoot = pallet_ethereum::IntermediateStateRoot<Self>;
+	type PostLogContent = PostBlockAndTxnHashes;
+	// Maximum length (in bytes) of revert message to include in Executed event
+	type ExtraDataLength = ConstU32<30>;
+}
+
+parameter_types! {
+	pub const DefaultYearlyInflation: Perbill = Perbill::from_perthousand(5);
+}
+
+pub struct IdentityAccountIdConvert;
+
+impl pallet_score_staking::AccountIdConvert<Runtime> for IdentityAccountIdConvert {
+	fn convert(account: AccountId) -> <Runtime as frame_system::Config>::AccountId {
+		account
+	}
+}
+
+impl pallet_score_staking::Config for Runtime {
+	type Currency = Balances;
+	type RuntimeEvent = RuntimeEvent;
+	type AccountIdConvert = IdentityAccountIdConvert;
+	type AdminOrigin = EnsureRootOrHalfCouncil;
+	// Temporary suspend of reward
+	type YearlyIssuance = ConstU128<0>;
```

Comment:
I suggest we set the constant already,  otherwise we'll need another upgrade to do it.

The pool is defaulted **not** to start.

Reply:
sure